docusaurus/docs/dev-docs/plugins/graphql.md

```diff
@@ -709,11 +727,11 @@ export default {
 
 </details>
 
-#### Middlewares
+##### Middlewares
 
-[Middlewares](/dev-docs/backend-customization/middlewares) can be applied to a GraphQL resolver through the `resolversConfig.[MyResolverName].middlewares` key.
+[Middlewares](/dev-docs/backend-customization/middlewares) can be applied to a GraphQL resolver through the `resolversConfig.[MyResolverName].middlewares` key. The only difference between the GraphQL and REST implementations is that the `config` key becomes `options`.  
```

Comment:
Yeah this one is on me, it should have been config since the beginning but now we have options :man_facepalming: